SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebuggerConfigUI.cpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialDebuggerConfigUI.h"
+#include "EngineClasses/SpatialNetDriver.h"
```

Comment:
Put a line break between these two, we always want the .h first, and I think clang-format might reorder these two

Reply:
Turns out it doesn't! Still gonna add the linebreak